### PR TITLE
LPS-196403 API Builder fields should be system

### PR DESCRIPTION
**Pull request by 4lejandrito**
https://liferay.atlassian.net/browse/LPS-196403.

This turns all fields from custom to system (except for relationship fields, which will be done in https://liferay.atlassian.net/browse/LPS-196412).

### CODE REVIEW

**Comment by 4lejandrito** [comment]
ci:test:relevant

**Comment by 4lejandrito** [comment]
ci:test:sf

File: modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/odata/entity/v1_0/ObjectEntryEntityModel.java [hunk 1]
```diff
@@ -249,7 +251,7 @@ private Map<String, EntityField> _getStringEntityFieldsMap(
 			).build();
 
 		for (ObjectField objectField : objectFields) {
-			if (objectField.isSystem()) {
+			if (!objectDefinition.isSystem() && objectField.isSystem()) {
```
Comment (4lejandrito) [inline modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/odata/entity/v1_0/ObjectEntryEntityModel.java:254]: Allow only modifiable object definitions, not service builder ones.

Comment (4lejandrito) [reply modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/odata/entity/v1_0/ObjectEntryEntityModel.java]: Fixed.

**Comment by 4lejandrito** [comment]
ci:forward:force

**Comment by 4lejandrito** [comment]
ci:forward

**Comment by ccorreagg** [comment]
ci:stop

**Comment by ccorreagg** [comment]
ci:forward

**Comment by 4lejandrito** [comment]
ci:forward:force

**Pull request closed without merging**